Document/0x06h-Testing-Platform-Interaction.md

```diff
@@ -2523,6 +2524,10 @@ onMatch:  <WKWebView: 0x1508b1200; frame = (0 0; 320 393); layer = <CALayer: 0x1
 hasOnlySecureContent:  false
 ```
 
+##### Testing for WebView URI Manipulation
+
+Make sure that the webview its URI cannot be manipulated by the user in order to load other types of resources than necessary for the functioning of the webview. This can be specifically dangerous the momen the webview resources come from the local file system, allowing the user to navigate to other resources within the application.
```

Comment:
```suggestion
Make sure that the WebView's URI cannot be manipulated by the user in order to load other types of resources than necessary for the functioning of the WebView. This can be specifically dangerous when the WebView's content is loaded from the local file system, allowing the user to navigate to other resources within the application.
```